src/components/Navbar/Navbar.style.scss

```diff
@@ -0,0 +1,20 @@
+.nav {
+}
+.nav-text {
+  display: flex;
+  height: 60px;
+  text-decoration: none;
+  color: white;
+  font-size: 25px;
+}
+li {
```

Comment:
avoid using native tags like li, div, ... prefer to use classnames, here put the code in .nav-text